python/ray/tests/test_multiprocessing.py

```diff
@@ -524,6 +540,31 @@ def f(args):
     with pytest.raises(StopIteration):
         result_iter.next()
 
+    # Repeat of the above with iterators as inputs
```

Comment:
Can we parametrize this test as well? (See [my earlier comment](https://github.com/ray-project/ray/pull/24117/files#r856668382)).

Reply:
Done!